api/pkg/transformer/pipeline/testdata/valid_table_join_multiple_columns.yaml

```diff
@@ -34,17 +34,29 @@ transformerConfig:
           steps:
             - sort:
                 - column: row_number
-                  order: DESC
+                  order: ASC
             - renameColumns:
                 row_number: rank
-                id: driver_id
-            - selectColumns: [driver_id, name, rank]
+                id: driver_id            
       - tableJoin:
           leftTable: driver_table
           rightTable: driver_feature_table
           outputTable: result_table
           how: LEFT
           onColumns: [driver_id, name]
+      - tableTransformation:
+          inputTable: result_table
+          outputTable: result_table
+          steps:
+            - updateColumns:
+              - column: is_weekend
+                expression: IsWeekend(driver_table.Col('eta'), 'Asia/Jakarta')
+              - column: day_of_week
+                expression: DayOfWeek(driver_table.Col('eta'), 'Asia/Jakarta')
+              - column: pretty_date
+                expression: ParseTimeStampsWithFormat(driver_table.Col('eta'), 'Asia/Jakarta', '2006-01-02')
```

Comment:
I think `ParseTimeStampsWithFormat` as function name is quite long. Wdyt about `FormatTimestamp` for function name?

Reply:
Aggree, I think `FormatTimestamp` is better name